cleanlab/latent_algebra.py

```diff
@@ -18,7 +18,7 @@
 # ## Latent Algebra
```

Comment:
Consider moving lower level helper functions like those in this `latent_algebra.py` and `util.py` files to a `utils/` subdirectory. 
In particular, we want to be careful about our documentation becoming cluttered by functions intended only for the most advanced users.  If all helper functions are in a `utils/` subdirectory, then the documentation website can hide their documentation inside a pull-down `utils` tab that presumably only advanced users will ever explore.

@weijinglok

Reply:
Note that several of the mathematical equalities provided in latent_algebra are pretty impressive results from our PhD work (some solved by friend Tailin Wu) and not all of them published. I agree this is not a core module and we could move it to utils (happy to make this change), but right now it does serve the purpose of showing of some impressive results for those familiar with the field and it will be harder for folks to stumble across those equalities if they move to a utils folder. Given that what do you think about moving `latent_algrebra` to a `utils/` folder now... versus waiting until we add more tasks and things start to get more cluttered?

Reply:
I would move it now to minimize API breaking changes going forward.  Especially if it's just intended for researchers, who you can still point to this functionality from the confidentlearning-reproduce repo README.

Unless there are useful things a data scientist can do based on the `latent_algebra` functionality?  If so, these use-cases would need to be made more clear in the docstrings for this to remain at the top-level and be something the average user can actually leverage.  Actually even if moving these to lower `utils/` level, it could be helpful to add one extra sentence to each docstring explaining what purpose of the function is (ie. why it might be useful).

Reply:
done